### DISCUSSION TITLE
I think login.js files path is wrong

### DISCUSSION THREAD

**Discussion by nabi-chan** [coder/code-server#2182] [Q&A] [answered]
Hello, I'm using `code-server` at ubuntu 18.04.

when user login...
login.html gets `login.js` files path `/dist/register.js`,
but real `login.js` path is `/dist/pages/login.js`

I want to know if this is my own problem.
Send picture together!

![image](https://user-images.githubusercontent.com/42037851/95584981-80040600-0a79-11eb-84d3-104cf681a37e.png)

**Comment by code-asher** [comment, marked as answer]
Yup, good catch! This is fixed in master but it's not in a release yet.
We should be getting a new release out soon.
